plugins/out_stackdriver/stackdriver.h

```diff
@@ -52,6 +52,7 @@
 #define DEFAULT_SEVERITY_KEY "logging.googleapis.com/severity"
 #define DEFAULT_TRACE_KEY "logging.googleapis.com/trace"
 #define DEFAULT_SPAN_ID_KEY "logging.googleapis.com/spanId"
+#define DEFAULT_TRACE_SAMPLED_KEY "logging.googleapis.com/trace_sampled"
```

Comment:
I think we've decided to make this `camelCase` on the Google side.
```
#define DEFAULT_TRACE_SAMPLED_KEY "logging.googleapis.com/traceSampled"
```
(I was going to make this a suggestion but I think you'll need to do it so you can make the right commit message on it)

Reply:
done